lib/rack/request.rb

```diff
@@ -16,7 +16,20 @@ class << self
       attr_accessor :ip_filter
     end
 
-    self.ip_filter = lambda { |ip| /\A127\.0\.0\.1\Z|\A(10|172\.(1[6-9]|2[0-9]|30|31)|192\.168)\.|\A::1\Z|\Afd[0-9a-f]{2}:.+|\Alocalhost\Z|\Aunix\Z|\Aunix:/i.match?(ip) }
+    VALID_IPV4_OCTET = '(25[0-5]|2[0-4][0-9]|[01]?[0-9]?[0-9])'.freeze
```

Comment:
No reason to make this a constant.  As I mentioned earlier, a local variable is better.  We want to avoid introducing new public comments for implementation details.

Reply:
Changed to local variable `valid_ipv4_octet`.